### ISSUE TITLE
[TDB] Forest doesn't report forks correctly

### ISSUE THREAD

**Issue by lemmih** [ChainSafe/forest#4825]
# Summary

TDB

# Requirements

TDB

# Motivation

TDB

**Comment by LesnyRumcajs** [comment]
@elmattic could you please assess if this issue is still valid?

**Comment by elmattic** [comment]
Yes, this is absolutely necessary and vital for certain Forest components, such as the message pool.

**Comment by LesnyRumcajs** [comment]
I mean - do you know if it's still an issue? Or was it resolved, e.g., via the new chain follower? I doubt it, but hope dies last!

**Comment by elmattic** [comment]
It’s still an issue. As far as I know, the new chain follower doesn’t have this feature.

Perhaps we can start by creating an issue to look into this and see how hard it would be to support.

**Comment by LesnyRumcajs** [comment]
> It’s still an issue. As far as I know, the new chain follower doesn’t have this feature.
> 
> Perhaps we can start by creating an issue to look into this and see how hard it would be to support.

@coderabbitai create an issue for investigating this.
